### ISSUE TITLE
Misalignment issues on different screen resolutions

### ISSUE THREAD

**Issue by mbechev** [telerik/kendo-angular#3739]
**Describe the bug**
The Scheduler has some misalignment issues caused by different screen resolutions (reproduced on mobile devices as well).

Month view issue (snapshot taken from the site):
![image](https://user-images.githubusercontent.com/33104481/184620410-53540ce0-4c23-4fdd-b3a5-5a036af9ea7e.png)

Week view all-day slot misalignment (snapshot taken from the site):
![image](https://user-images.githubusercontent.com/33104481/184620019-f79975cc-6511-401a-8e2d-3436953cd857.png)

**To Reproduce**
Zoom the browser
https://stackblitz.com/edit/angular-q4aaeu?file=src%2Fapp%2Fapp.component.ts

**Comment by HaberkornJonas** [comment]
I face the same issue.
I noticed that it happens as soon as I set a height via CSS to the <kendo-scheduler>

**Comment by Leafgard** [comment]
Please also note that there is another misalignment issue when having multiple lines in header, as you can see below:

![image](https://user-images.githubusercontent.com/25567034/187849035-b1fcbbb9-656f-4d2b-9d76-fb4d678e5c97.png)

```html
<kendo-scheduler
    scrollTime="08:00"
    [weekStart]="weekStart"
    [kendoSchedulerBinding]="events"
    [selectedDate]="selectedDate"
    [selectedViewIndex]="selectedViewIndex"
    [showWorkHours]="true"
  >
    <ng-template kendoSchedulerDateHeaderTemplate let-date="date">
      <span>{{ date | kendoDate:'EEEE' }}</span>
      <br>
      <span>{{ date | kendoDate:'dd.MM.yyyy' }}</span>
    </ng-template>
    <kendo-scheduler-day-view> </kendo-scheduler-day-view>
    <kendo-scheduler-week-view> </kendo-scheduler-week-view>
    <kendo-scheduler-month-view> </kendo-scheduler-month-view>
    <kendo-scheduler-timeline-view> </kendo-scheduler-timeline-view>
    <kendo-scheduler-agenda-view> </kendo-scheduler-agenda-view>
  </kendo-scheduler>
```

In my opinion this issue should have **higher severity**.

Also, none of these issues are happening on the Kendo UI React library. Perhaps the structure of the Scheduler is different? If so, it might be a good idea to **build Kendo UI libraries atomically** to avoid such surprises.

**Comment by mbechev** [comment]
An example of the last reported issue:

https://stackblitz.com/edit/angular-cfkeca-zwqudl

**Comment by ipeshev** [comment]
Fix available in [latest dev version](https://www.telerik.com/kendo-angular-ui/components/installation/development-builds/).

**Comment by stnikolova** [comment]
Fix available in **v4.3.5** and above of **@progress/kendo-angular-scheduler**.
